Tools.json sorting pt. 8

RarkGrames opened the pull request:
#### Summary
```SUMMARY: Infrastructure "Sorted out tools.json into several category files"```

#### Purpose of change
Sorting out tools.json into separate category files in json\items\tool for easier handling.
Continuation of #37218.

#### Describe the solution
- Doing woodworking.json and workshop.json

#### Testing
Game loads properly. Double checked every item to be sure.

ZhilkinSerg commented:
I find this line count difference strange for only 3 change files and 2 new files:

![image](https://user-images.githubusercontent.com/16213433/72765169-3712d580-3bfc-11ea-97eb-55ed88caa7c7.png)

Expected difference would be 4 lines (for square brackets in new files).

RarkGrames commented:
> 
> 
> I find this line count difference strange for only 3 change files and 2 new files:
> 
> ![image](https://user-images.githubusercontent.com/16213433/72765169-3712d580-3bfc-11ea-97eb-55ed88caa7c7.png)
> 
> Expected difference would be 4 lines (for square brackets in new files).

Somehow a couple of entries managed to stay in from the woodworking and workshop, probably from merge conflicts.
Also I added one more file for musical instruments that I missed earlier, and deleted the original tools.json.
